### ISSUE TITLE
Debug Argmax

### ISSUE THREAD

**Issue by Jalen1** [Jalen1/Retinal-Disease-Classification#8]
When training the model, our code currently finds the classification assigned to a given image by calculating the argmax of the image's label:
- This returns the index of the greatest number in that image's label tensor.
- Each label consists of 47 integers contained in a tensor, 0 indicating negative and 1 indicating positive for disease. 

Currently, the argmax function returns the first occurrence of the greatest number in each label, which always happens to be the ID of the given image, which goes from 0 up to 1920 for our training set.

**TODO:**

- [x] Truncate the input label columns by removing the "ID" column and the "Disease_Risk" columns (the two leftmost columns in the .csv files that correspond with the training/test sets.)
- [x] Fix prediction function by calculating the correct label for a given image when running argmax
